### ISSUE TITLE
Any comments and refinements for WOW Schedule?

### ISSUE THREAD

**Issue by zee-moz** [mozillascience/working-open-workshop#13]
@acabunoc @arlissc @kaythaney  @auremoser 
@stephwright

**Comment by stephwright** [comment]
Is the elevator pitch still going to be part of the Work Week?  Or is it in there and I'm just missing it?

**Comment by zee-moz** [comment]
@stephwright do you mean the fellows workweek, or WOW?
wow schedule is here: https://github.com/mozillascience/working-open-workshop/blob/master/wow-schedule.md

**Comment by stephwright** [comment]
Oops... Fellows WW... NM :flushed:

**Comment by kaythaney** [comment]
@auremoser had suggested them talking at hacks/hackers as a way to weave this in.

sent on the move. apologies for typos.

**Comment by auremoser** [comment]
Yes, I thought we could work on their pitches and then have them test an abbreviated version at Hacks/Hackers. Or we might just make that a community event and we'll do a more generic presentation about what the science lab does, and what fellows do for us, and how that all could potentially related to journalism.

Here's [the fellows workweek schedule](https://public.etherpad-mozilla.org/p/wow-mslparty). I moved some things around as @zee-moz suggested that we should test WOW curriculum with them earlier in the week so we have time to edit. I tried to squeeze in some project pitch work that Tuesday as well so they'll be prepped for Hacks Hackers as a dry-run for their mixer talks.

Let me know if you have comments on this as well: https://public.etherpad-mozilla.org/p/wow-mslparty
